tokenomics-explainers/002-rework-intro.md

```diff
@@ -44,10 +49,21 @@ Node-level penalties and forced validator exits are also included in the rework
 
 Megapools will enable multiple validators to share the same Ethereum withdrawal address – this **reduces initial and ongoing gas costs** for node operators.
 
-**RPL inflation will be reduced from 5% to 1.5%**, because the current 3.5% inflation used for RPL rewards to NOs is no longer necessary.
+**RPL issuance will be reduced from 5% to 1.5%**, because the current 3.5% issuance used for RPL rewards to NOs is no longer necessary.
 
 Forced delegate upgrades (with a grace period) will limit the technical debt the Rocket Pool protocol accumulates from previous versions.
 
+## Personas
+
+|Persona|Now|With Rework|
+|-|-|-|
+|**Existing Rocket Pool Node Operator**|I need RPL to create minipool validators. I have to remain collateralized at an RPL staked value of at least 10% of my borrowed ETH to receive RPL issuance rewards. My commission is 14% of my borrowed ETH.|I do not need RPL to create megapool validators. My commission is 3.5% of my borrowed ETH. If I choose to stake RPL I will receive additional ETH rewards (5% of protocol revenue from all megapools) proportional to my share of staked RPL (up to a maximum). I do not need to maintain a minimum stake to receive these rewards. I will benefit from value accrual to RPL (5.5% of protocol revenue from all megapools) or a 5.5% increase to the rewards for staking RPL. I will no longer receive rewards in RPL.
```

Comment:
```suggestion
|**Existing Rocket Pool Node Operator**|I need RPL to create minipool validators. I have to remain collateralized at an RPL staked value of at least 10% of my borrowed ETH to receive RPL issuance rewards. My commission is 14% of my borrowed ETH.|I do not need RPL to create megapool validators. My commission is 3.5% of my borrowed ETH. If I choose to stake RPL I will receive additional ETH rewards (5% of borrowed ETH revenue from all megapools) proportional to my share of staked RPL. I do not need to maintain a minimum stake to receive these rewards. I will benefit from value accrual to RPL (5.5% of borrowed ETH revenue from all megapools) or from this revenue being paid to staked RPL. I will no longer receive rewards in RPL.
```

Reply:
Sorry for more changes... maybe we simplify by removing "up to a maximum" since "proportional" is vague enough to cover that (or you could keep it in there)? I realized "protocol revenue" is really just the 14% fee (not total borrowed ETH revenue since 86% of it belongs to rETH holders), so 5% of "protocol revenue" would only be 5% of the 14%. Similarly, a 5.5% increase could come across as increasing your rewards by 5.5% which isn't much. Instead it's 5.5% of borrowed ETH revenue also being paid to staked RPL.

I think this one is the most challenging to condense